### ISSUE TITLE
[Feature request]:NES.Emu Mapper Update

### ISSUE THREAD

**Issue by TheBoxGame** [Rakashazi/emu-ex-plus-alpha#1461]
### Describe the feature

@Rakashazi add this mappers to nes.emu 256 419 313 502 295 416 plis and add display nsf 
![NSF UI](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/724918ea-9f8f-4cd3-813e-079de10aa958)

**Comment by ErickN13** [comment]
From version 1.5.72 onwards, the NES.EMU can play NSF files, but it still does not have a display like the one in the Screenshot, NOTE: The NES.emu emu opens NSF files, but when it is playing The NSF, the screen goes black😐

**Comment by TheBoxGame** [comment]
yes, that's true and it would be useful, and what about mappers, I mean a few roms that I would like to run

**Comment by TheBoxGame** [comment]
![256146911-248bca5e-e976-479e-9602-ba1cbb05546c](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/85706f9e-722f-4909-af9c-ef8002177e76)
![256146893-43bef542-8b25-4570-bdbc-56205e6c588c](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/92f5d211-ff4a-43f9-a7cc-7e17488e853e)
![256146904-12fe4c0a-3932-4816-bd61-ff493e83cc2a](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/85c072a1-4fc9-4cd8-a3f6-df3e70f5ed03)
![259689248-4dec1881-40d3-4b37-97e2-cfc433773000](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/b3641004-6430-4690-889c-dec9b1ac55d0)
![XBF860MENU](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/5792dc45-de22-44c4-9c4c-6e1831fbfd5b)
![GENERICS2023A](https://github.com/Rakashazi/emu-ex-plus-alpha/assets/109180532/98a8b486-9dc3-4edb-ab62-74fd2b9eb708)

**Comment by ErickN13** [comment]
>  yes, that's true and it would be useful, and what about mappers, I mean a few roms that I would like to run

But NES.emu runs any file that has the [.NES] extension and that is compatible with the NES Core

**Comment by ErickN13** [comment]
What do you mean by that?

**Comment by TheBoxGame** [comment]
these roms don't have these mappers because I write error loading rom, for example 50-in-1 has mapper 256 and 419 and when you enter this rom you won't run it because it has mapper mapper ines 2.0

**Comment by TheBoxGame** [comment]
here is a list of mappers from these roms plug n play 50-in-1 racing 
mapper 256 and 419
Block 14-in-1 Mapper 295
Super Game 10-in-1 YG-02 Mapper 502 
Super Mario Family Mapper 353

**Comment by ErickN13** [comment]
Oh okay, bye

**Comment by TheBoxGame** [comment]
ok good bye

**Comment by ErickN13** [comment]
What is your email? Let's talk there

**Comment by TheBoxGame** [comment]
ok please theboxgamepl@gmail.com
